SimpleRabbit.NetCore/Service/ParallelMessageHandler.cs

```diff
@@ -70,7 +70,9 @@ public bool Process(BasicMessage message)
                     var tailTask = ContinueTaskQueue(task, message, key, item);
                     _tasks[key] = tailTask; // add completed task to the to be used.
                 }
-                return false;
+                
+                // Ignoring as we are handling the acking ourselves in parallel
+                return Acknowledgement.Manual;
```

Comment:
Is this `Acknowledgement.Ignore`?

Reply:
We've changed the naming to `Acknowledgement.Manual`, to better reflect the reason on why you would want to return this enum value (because you are manually acknowledging the message yourself in your IMessageHandler implementation)